.travis.yml

```diff
@@ -20,14 +20,19 @@ matrix:
     # JOB 2
     - env: CF_VERSION=master
       install:
+        - sudo apt install -y software-properties-common
+        - sudo add-apt-repository -y ppa:deadsnakes/ppa
```

Comment:
what is deadsnakes? I know about ppa, but this is a bit obscure, might be nice to comment

Reply:
I amended the commit message to mention deadsnakes.